src/preferences/upgrade.cpp

```diff
@@ -329,6 +385,29 @@ UserSettingsPointer Upgrade::versionUpgrade(const QString& settingsPath) {
         configVersion = "1.9.0";
         config->set(ConfigKey("[Config]","Version"), ConfigValue("1.9.0"));
     }
+
```

Comment:
this is quite early in the function. Meaning that the rest of the function body seems to be sorted in ascending version order and this new section is not sorted that way. Can somebody with the upgrade procedure check whether this is correct here?

Reply:
Yes, this wasn't very clear to me either. I double-checked and I see that I have to move it a bit lower.

But it has to go before the block 
```
if (configVersion.startsWith("1.11")) {
```
because at the end of that block the version in the config is set to the current version.
```
config->set(ConfigKey("[Config]", "Version"), ConfigValue(VersionStore::version()));
```

I will leave a comment explaining this.

Reply:
Ok, I would say it's correct now. Thanks @Swiftb0y for noticing that it looked a bit fishy, after studying the code a bit more I now think I understand the intent and I added a comment. But it would indeed be good if someone who knows this code better can confirm.

Reply:
Yeah, judging by the blame, none of the actual changes have been made by people that are still active...

Reply:
I could at least take a look but the I'll be afk for a few more days